recipes/revbayes/build.sh

```
@@ -1,53 +1,50 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
+export INCLUDES="-I${PREFIX}/include"
+export LIBPATH="-L${PREFIX}/lib"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+export CXXFLAGS="${CXXFLAGS} -O3 -I${PREFIX}/include"
+
+mkdir -p $PREFIX/bin
 cd projects/cmake
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)